guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -59,64 +59,8 @@ Ensure that you enter the full path to the `.tar` file.
 ifndef::orcharhino[]
 For more information, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
 endif::[]
-. On all hosts registered to your {SmartProxyServer}, enter the following commands to reinstall the bootstrap RPM, reregister clients, and refresh their subscriptions.
-+
-You can use the remote execution feature to perform this step.
-For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting up Remote Jobs] in _{ManagingHostsDocTitle}_.
-ifndef::satellite,orcharhino[]
-+
-On hosts running {EL} 8+, run the following commands:
-endif::[]
-+
-ifndef::orcharhino[]
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# {client-package-remove-el8} katello-ca-consumer*
-
-# {client-package-install-el8} http://_new-{smartproxy-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm
-
-# subscription-manager register \
---environment="_My_Lifecycle_Environment_" \
---force \
---org="_My_Organization_"
-
-# subscription-manager refresh
-----
-endif::[]
-ifdef::orcharhino[]
-ifeval::["{client-pkg-ext}" == "rpm"]
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# {client-package-remove} katello-ca-consumer*
-
-# {client-package-install} http://_new-{smartproxy-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm
-
-# subscription-manager register \
---environment="_My_Lifecycle_Environment_" \
---force \
---org="_My_Organization_"
-
-# subscription-manager refresh
-----
-endif::[]
-ifeval::["{client-pkg-ext}" == "deb"]
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# wget http://_new-{smartproxy-example-com}_/pub/katello-rhsm-consumer
-
-# chmod +x katello-rhsm-consumer
-
-# ./katello-rhsm-consumer
-
-# subscription-manager register \
---environment="_My_Lifecycle_Environment_" \
---force \
---org="_My_Organization_"
-
-# subscription-manager refresh
-----
-endif::[]
-endif::[]
+. Re-register all hosts that are registered to your {SmartProxyServer}.
```

Comment:
Very small nit-pick. Our downstream guidelines say this about hyphenated prefixes: 

"In most cases, do not use a hyphen to connect a prefix or suffix to a word. There are cases where a hyphen between a prefix and root word adds clarity. For more information, see ..."

We do not yet have a Vale rule to flag "pre-", "post-", "re-", etc. I think this might be a good rule to add. :-)

Reply:
Correction: We do have a Vale rule. It does not include "re-register", so I created a [PR](https://github.com/redhat-documentation/vale-at-red-hat/pull/852) to fix this.

Reply:
Applied.